.flowconfig

```diff
@@ -0,0 +1,15 @@
+[ignore]
+<PROJECT_ROOT>/bin/.*
+<PROJECT_ROOT>/coverage/.*
+<PROJECT_ROOT>/gui/.*
+<PROJECT_ROOT>/lib/.*
+<PROJECT_ROOT>/test/.*
+<PROJECT_ROOT>/tmp/.*
+
+<PROJECT_ROOT>/node_modules/jsverify/.*
```

Comment:
According to the `flow ls` output, it doesn't seem to ignore anything... (event images and json files are listed)

Since I didn't know if flow needed to analyse node modules, and it didn't seem to be an issue (except maybe for performance, but it seems ok even with modules), I decided not to ignore them, and see whether it became an issue.